bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/YIOremoteBindingConstants.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.yioremote.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link YIOremoteBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Michael Loercher - Initial contribution
+ */
+@NonNullByDefault
+public class YIOremoteBindingConstants {
+
+    private static final String BINDING_ID = "yioremote";
+
+    // List of all used global variables
+    public static enum YIOREMOTEHANDLESTATUS {
```

Comment:
Please use underscores to separate the words.
```suggestion
    public static enum YIO_REMOTE_HANDLE_STATUS {
```